In PLD Linux, browser-plugins framework symlinks enabled plugins to plugins/ dir
access directly from there, instead of symlinking in google-chrome.spec

--- chromium-browser-20.0.1132.57/./src/chrome/common/chrome_paths.cc~	2012-07-13 21:13:51.000000000 +0300
+++ chromium-browser-20.0.1132.57/./src/chrome/common/chrome_paths.cc	2012-07-13 21:14:41.799431216 +0300
@@ -255,9 +255,11 @@
       cur = cur.Append(chrome::kPepperFlashPluginFilename);
       break;
     case chrome::FILE_PDF_PLUGIN:
-      if (!GetInternalPluginsDirectory(&cur))
+      if (!PathService::Get(base::DIR_MODULE, &cur))
         return false;
-      cur = cur.Append(kInternalPDFPluginFileName);
+
+      cur = cur.Append(FILE_PATH_LITERAL("plugins"))
+               .Append(kInternalPDFPluginFileName);
       break;
     case chrome::FILE_NACL_PLUGIN:
       if (!GetInternalPluginsDirectory(&cur))
